Packages/Testing-MIES/UTF_Utils.ipf

```diff
@@ -2282,3 +2282,103 @@ Function TextWaveToListWorks5()
 	CHECK_EQUAL_STR(list, refList)
 End
 /// @}
+
```

Comment:
Is an 8:1 test function to "feature" function typical?

Reply:
@timjarsky I'm not sure "typical" is a good word for it. I would say though that it does test the function behaviour completely enough. It is good practive to have short test cases, so that you know when a test fails what the problem is.